RELEASE_PROCESS.md

```diff
@@ -30,38 +30,86 @@ In order to release, you will need the following accounts/permissions:
 ### Informing everyone of a pending release
 
 - [ ] Announce your intention to drive the release to #cft-announce and #chef-server on slack.
+CLARIFICATION from @tas:
+    we can probably move the cft announce to be post promote
+    just copy the discourse post link to that channel to make sure folks see it
+    TODO: get the discourse link he was referring to.
 
 ### Getting the build to be released into current with a minor/major version bump
 
-- Update the version file to the version to be released if minor version needs to be bumped rather than just the patch.
+- Create a new branch from the latest master branch ? <----
+- Update the contents of the file VERSION to the version to be released if minor version needs to be bumped rather than just the patch [what about major version?].
+For updating just the patch verion, [insert 'skip this section, and' ?] refer the the section below on #Preparing for the release
 - Update the release notes with the version, date and context.
-- Run all the tasks for update from FrequentTasks.doc
-- Make sure the omnibus build is into current channel.
+QUESTION: here? https://github.com/chef/chef-server/wiki/Pending-Release-Notes
+- Update CHANGELOG.md ? <-----
+- Run all the tasks for update from dev-docs/FrequentTasks.md. [i need a walkthrough of this] <-----
+QUESTION: is this just updating ruby Gemfile.lock and erlang rebar.lock files? just delete the locks and regenerate? no updating of particular deps by hand?
+QUESTION: just make a new branch off of master to do this?
```

Comment:
this happens automatically due to expeditor pipeline but it would be useful to verify that the build to be promoted is actually present.

Reply:
Please clarify:

1) Expeditor pipeline creates a branch (yes/no)?
2) What is the name of the branch? (master?)
3) Where and/or how do we verify that the build to be promoted is actually present?

If some of the questions seem dumb, I apologize.  But I actually want this to be a 'dumb' step-by-step process where someone who doesn't know what he is doing can just sit down and dumbly follow steps.